### PR TITLE
Sorting and Commenting Rebel Templates

### PR DESCRIPTION
**Pull request by PoweredByPot**
- moved rebel groups creation to rebel templates
- removed unused 'lampsSDK' array
- removed 'sniperRifle' variable used to give petros a specifc weapon. He now uses a random 'unlockedRifles' weapon. Also removed the SP difficulty tweak that gave the player his rifle on easy difficulty.
- removed special unlocks for SP that only existed in vanilla rebel templates
- removed duplicate declaration of 'unlockedRifles' in initVar
- moved civillian vehicles arrays modifications to initVar
- added missing declaration of 'initialRifles' to initVar
- used append instead of a+b for adding arrays of items to previously declared arrays

### CODE REVIEW

File: A3-Antistasi/Templates/teamPlayerBlu3CBUN.sqf [hunk 1]
```diff
@@ -27,44 +52,52 @@ vehSDKTruck = "UK3CB_UN_B_V3S_Closed";
...
-supportStaticsSDKB2 = "RHS_DShkM_TripodHigh_Bag";
-AAStaticSDKB = "I_AA_01_weapon_F";
-MortStaticSDKB = "RHS_Podnos_Gun_Bag";
-supportStaticsSDKB3 = "RHS_Podnos_Bipod_Bag";
 
+//Civilian Vehicles
 civCar = "UK3CB_CHC_C_Ikarus";
 civTruck = "UK3CB_CHC_C_V3S_Recovery";
-civHeli = "UK3CB_UN_B_UH1H_MED";
+civHeli = "UK3CB_CHC_C_Mi8AMT";
```
[the remark this answers is not in the document]
Comment (PoweredByPot) [reply A3-Antistasi/Templates/teamPlayerBlu3CBUN.sqf]: yes, it was calling a blufor heli, which I don't think will support undercover properly. We also dont use blufor vehicles for civ slots on the other BLUFOR rebels template

[the remark this answers is not in the document]
Comment (SonorousMeerkat) [reply A3-Antistasi/Templates/teamPlayerBlu3CBUN.sqf]: Technically afaik it should, either way a huey is used on one of the missions rhs. can't remember which though

[the remark this answers is not in the document]
Comment (Spoffy) [reply A3-Antistasi/Templates/teamPlayerBlu3CBUN.sqf]: So do we keep the change to Mi8, or switch back to the Huey? Your call Meerkat. Either should work for undercover. And if it doesn't, that's a bug.

[the remark this answers is not in the document]
Comment (SonorousMeerkat) [reply A3-Antistasi/Templates/teamPlayerBlu3CBUN.sqf]: keep as mi-8 for now. I'll take a proper pass over the heli's to find a suitable one. (Kinda wish we could select random it tbf, i think i like variety a bit too much...)

**Pull request merged**